modules/extension/wmts/src/main/java/org/geotools/data/wmts/model/WMTSCapabilities.java

```diff
@@ -0,0 +1,537 @@
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.data.wmts.model;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.logging.Logger;
+
+import org.eclipse.emf.common.util.EList;
+import org.geotools.data.ows.CRSEnvelope;
+import org.geotools.data.ows.Capabilities;
+import org.geotools.data.ows.Layer;
+import org.geotools.data.ows.OperationType;
+import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.referencing.CRS;
+import org.opengis.referencing.FactoryException;
+import org.opengis.referencing.crs.CoordinateReferenceSystem;
+import org.opengis.referencing.cs.AxisDirection;
+import org.opengis.referencing.operation.TransformException;
+
+import com.vividsolutions.jts.geom.Coordinate;
+import com.vividsolutions.jts.geom.GeometryFactory;
+
+import net.opengis.ows11.WGS84BoundingBoxType;
+import net.opengis.ows11.AllowedValuesType;
+import net.opengis.ows11.BoundingBoxType;
+import net.opengis.ows11.CodeType;
+import net.opengis.ows11.DCPType;
+import net.opengis.ows11.DomainMetadataType;
+import net.opengis.ows11.DomainType;
+import net.opengis.ows11.LanguageStringType;
+import net.opengis.ows11.OperationsMetadataType;
+import net.opengis.ows11.RequestMethodType;
+import net.opengis.ows11.ValueType;
+import net.opengis.wmts.v_1.CapabilitiesType;
+import net.opengis.wmts.v_1.ContentsType;
+import net.opengis.wmts.v_1.DimensionType;
+import net.opengis.wmts.v_1.LayerType;
+import net.opengis.wmts.v_1.TileMatrixLimitsType;
+import net.opengis.wmts.v_1.TileMatrixSetLimitsType;
+import net.opengis.wmts.v_1.TileMatrixSetLinkType;
+import net.opengis.wmts.v_1.TileMatrixSetType;
+import net.opengis.wmts.v_1.TileMatrixType;
+import net.opengis.wmts.v_1.URLTemplateType;
+import org.geotools.data.wms.xml.Dimension;
+import org.geotools.data.wms.xml.Extent;
+
+/**
+ * Represents a base object for a WMTS getCapabilities response.
+ *
+ * (Based on existing work by rgould for WMS service)
+ * @author ian
+ * @author Emanuele Tajariol (etj at geo-solutions dot it)
+ *
+ */
+public class WMTSCapabilities extends Capabilities {
+
+    static public final Logger LOGGER = org.geotools.util.logging.Logging
+            .getLogger("org.geotools.data.wmts");
+
+    private static CoordinateReferenceSystem CRS84;
+    static {
+        try {
+            CRS84 = CRS.decode("CRS:84");
+        } catch (FactoryException e) {
```

Comment:
Btw, if you want a WGS84 lon/lat oriented, DefaultGeographicCRS.WGS84 should do the trick (hopefully)